src/server/game/Chat/Chat.cpp

```diff
@@ -310,12 +310,16 @@ bool ChatHandler::ExecuteCommandInTable(std::vector<ChatCommand> const& table, c
                 uint32 zoneId = player->GetZoneId();
                 std::string areaName = "Unknown";
                 std::string zoneName = "Unknown";
+                int locale = GetSessionDbcLocale();
```

Comment:
minor thing, you can even use "auto" instead of "int", it will take the return type of GetSessionDbcLocale

Reply:
that or specify the integer size please